cmd/validate/image.go

```diff
@@ -42,7 +42,7 @@ import (
 	validate_utils "github.com/enterprise-contract/ec-cli/internal/validate"
 )
 
-type imageValidationFunc func(context.Context, app.SnapshotComponent, policy.Policy, []evaluator.Evaluator, bool) (*output.Output, error)
+type imageValidationFunc func(context.Context, app.SnapshotComponent, policy.Policy, []evaluator.Evaluator, bool, []app.SnapshotComponent) (*output.Output, error)
```

Comment:
Could pass in the whole snapshot spec:
```suggestion
type imageValidationFunc func(context.Context, app.SnapshotComponent, app.SnapshotSpec, policy.Policy, []evaluator.Evaluator, bool) (*output.Output, error)
```